netci.groovy

```diff
@@ -235,6 +235,30 @@ commitPullList.each { isPr ->
   }
 }
 
+// True when this is a PR job, false for commit. We generally don't want to run the Loc status
+// check on PRs (as we don't want to block PRs due to unlocalized resources) and never want to
+// bother running it on feature branches.
+def locStatusCommitPullList = [false]
+if (branchName.startsWith("features/")) {
+  locStatusCommitPullList = []
+}
+
+// Loc status check
+locStatusCommitPullList.each { isPr ->
+  def jobName = Utilities.getFullJobName(projectName, "loc_status", isPr)
```

Comment:
Won't this still run on commit in feature branches? That would mean a job is running and failing all the time.

Reply:
@jaredpar See lines 242 and 243, where I check if `branchName` starts with "features" and set `locStatusCommitPullList` to the empty set. I think that will prevent it from running in feature branches at all (but please let me know if I'm missing something).